### DISCUSSION TITLE
[shape_infer] Can we use one PR to add shape_infer() for multiple OPs?

### DISCUSSION THREAD

**Discussion by usstq** [usstq/openvino#4] [Q&A] [answered]
Currently one Pull Request only contain one OP, which makes both rebase & CI time-consuming (each rebase means a lot of files to re-compile), if we can merge multiple OPs (from same the contributor) into one PR, then it will be much more efficient.

**Comment by e-nugmanova** [comment, marked as answer]
I don't mind if we will have several simple ops in one PR
